components/Footer.tsx

```diff
@@ -1,70 +1,94 @@
 import Image from "next/image";
 import { env } from "@/env.mjs";
+import { FaYoutube, FaLinkedin, FaGithub, FaEnvelope } from "react-icons/fa";
+
+const SocialLink = ({ href, icon: Icon, label }) => (
+  <a
+    href={href}
```

Comment:
Let's open links in new tab and also add appropriate `rel` values

```suggestion
    href={href}
    target="_blank"
    rel="noopener noreferrer"
```